### ISSUE TITLE
Button URL doesn't update on client

### ISSUE THREAD

**Issue by TheodoreHua** [qwertyquerty/pypresence#162]
I haven't fully tested this, but it seems like a caching issue? 

In this situation, I have a button with a URL set on it. Clicking said button works just fine, but on the next update, the URL changes. However when I try to click the button in my Discord client, the URL remains the same until a refresh is done locally (Ctrl + R on the Discord client). I've tried this on mobile, desktop, and browser to the same result. Is this a Discord issue?

**Comment by TheodoreHua** [comment]
Closing. Seems to be an instance of Discord lagging for a small bit.

**Comment by TheodoreHua** [comment]
More testing was done, it was not in fact Discord lag. The issue doesn't seem to happen on mobile (that was lag), but it's reproducible consistently on desktop/browser.

**Comment by Hunter2807** [comment]
> More testing was done, it was not in fact Discord lag. The issue doesn't seem to happen on mobile (that was lag), but it's reproducible consistently on desktop/browser.

I mean, pypresence doesn't even work for mobiles and browsers bro...

**Comment by TheodoreHua** [comment]
Let me rephrase, it's running on desktop, with Discord. That'll normally
show the RPC on all clients, said issue exists on browser and mobile.

**Comment by Hunter2807** [comment]
So you mean it updates the presence normally on the Desktop Client but doesn't update on Browser and Mobile?

**Comment by TheodoreHua** [comment]
It updates the presence normally on mobile (every time presence is updated, the link gets updated properly), but it doesn't work properly on either desktop or browser (link remains the same until a forced refresh [Ctrl+R])

**Comment by Hunter2807** [comment]
It's a discord issue then, as long as it's updating on at least one platform normally. This shows that the Discord API is at least *getting* the new presence information. How the clients handle it depends on the backend coding, not on pypresence 🤷‍♂️

**Comment by TheodoreHua** [comment]
Yeah, I figured.

**Comment by TheodoreHua** [comment]
Thanks anyway
